src/validate-binding-behavior-base.ts

```diff
@@ -31,24 +34,46 @@ export abstract class ValidateBindingBehaviorBase {
     controller.registerBinding(binding, target, rules);
     binding.validationController = controller;
     const trigger = this.getValidateTrigger(controller);
-    // tslint:disable-next-line:no-bitwise
-    if (trigger & validateTrigger.change) {
+    const event =
+      (trigger & validateTrigger.blur) === validateTrigger.blur ? 'blur'
+        : (trigger & validateTrigger.focusout) === validateTrigger.focusout ? 'focusout'
+          : null;
+    const hasChangeTrigger = (trigger & validateTrigger.change) === validateTrigger.change;
+    binding.isDirty = !hasChangeTrigger;
+    binding.validatedOnce = hasChangeTrigger && event === null;

```

Comment:
```suggestion
    // validatedOnce is used to control whether controller should validate upon user input
    //
    // always true when validation trigger doesn't include "blur" event (blur/focusout)
    // else it will be set to true after (a) the first user input & loss of focus or (b) validation
    binding.validatedOnce = hasChangeTrigger && event === null;
```
While I can understand the PR description, tracing back to PR to understand this piece of a little bit cryptic code isn't quick.